lib/newrelic_httprb/wrappers.rb

```diff
@@ -1,24 +1,5 @@
 module NewRelicHTTP
-  class HTTPResponse
-    attr_reader :response
-
-    def initialize(response)
-      @response = response
-    end
-
-    def [](key)
-      response.headers.each do |k,v|
-        if key.downcase == k.downcase
-          return v
-        end
-      end
-      nil
-    end
-
-    def to_hash
-      response.headers
-    end
-  end
+  HTTPResponse = NewRelic::Agent::HTTPClients::HTTPClientResponse
```

Comment:
If we depend on that specific interface, despite you are assuming it is stable, I would say we can add some tests to make sure it is/will be as we are expecting. I'm not sure if this is already implicit in the existing tests. If it is, you can discard my comment 😄